Artifacts.toml

```diff
@@ -4,5 +4,5 @@ os = "linux"
 lazy = true
 
     [[gurobilinux64.download]]
-    url = "https://packages.gurobi.com/10.0/gurobi10.0.0_linux64.tar.gz"
+    url = "https://packages.gurobi.com/11.0/gurobi11.0.0_linux64.tar.gz"
     sha256 = "91a9ce1464f5f948809fcdfbdeb55f77698ed8a6d6cfa6985295424b6ece2bd4"
```

Comment:
Tests still still ran `Gurobi Optimizer version 10.0.0 build v10.0.0rc2 (linux64)` because the content hash didn't change. 

We need to update the SHA etc. See https://github.com/jump-dev/Gurobi.jl/pull/490

Reply:
```Julia
run(`wget https://packages.gurobi.com/11.0/gurobi11.0.0_linux64.tar.gz`)
using Tar, Inflate, SHA
filename = "gurobi11.0.0_linux64.tar.gz"

julia> println("sha256: ", bytes2hex(open(sha256, filename)))
sha256: 6a1ec7499b230aea0542bc893bf0642ae8ce983dd5ef0c37cb3a253d827ce634

julia> println("git-tree-sha1: ", Tar.tree_hash(IOBuffer(inflate_gzip(filename))))
git-tree-sha1: 79f6322f611b73845e645125d5a54f17b16f6428
```

Reply:
Thanks, that was a fairly obvious oversight on my part ...